OutOfSchool/OutOfSchool.Tests/OrganizationControllerTest.cs

```diff
@@ -1,33 +1,280 @@
+
+            // Act
+            var response = await organizationController.GetOrganizationById(organizationId);
+
+            // Assert
+            Assert.IsInstanceOf<BadRequestObjectResult>(response);
         }
 
         [Test]
-        public void Test1()
+        public async Task Update_Organization_ReturnOkObjectResult()
         {
-            CreateOrganizationController.TestOk();
-            Assert.Pass();
+            // Arrange
+            var organization = new OrganizationDTO
+            {
+                Id = 0b101,
+                Title = "Title",
+                Website = "Website",
+                Facebook = "�������",
+                Instagram = "���������",
+                Description = "ererer",
+                MFO = "123456",
+                EDRPOU = "12345678",
+                INPP = "1234567891",
+                Type = 0,
+                UserId = "123"
+            };
+            
+            organizationService.Setup(it => it.Update(organization)).Returns(Task.FromResult(organization));
+
+            // Act
+            var result = await organizationController.Update(organization);
+
+            // Assert
+            Assert.IsInstanceOf<OkObjectResult>(result);
         }
+
+        [Test]
+        public async Task Update_OrganizationNotValidModel_ReturnsBadRequestObjectResult()
+        {
+            // Arrange
+            var organization = new OrganizationDTO
+            {
+                Id = 0b101,
+                Title = "Title",
+                Website = "Website",
+                Facebook = "�������",
+                Instagram = "���������",
+                Description = "ererer",
+                MFO = "123456",
+                EDRPOU = "12345678",
+                INPP = "1234567891",
+                Type = 0,
+                UserId = "123"
+            };
+
+            organizationController.ModelState.AddModelError("fakeError", "fakeError");
+
+            organizationService.Setup(it => it.Update(organization)).Returns(Task.FromResult(organization));
+
+            // Act
+            var response = await organizationController.Update(organization);
+
+            // Assert
+            Assert.IsInstanceOf<BadRequestObjectResult>(response);
+        }
+        [Test]
+        public async Task Update_OrganizationThatNull_ReturnsBadRequestObjectResult()
+        {
+            // Arrange
+            OrganizationDTO organization = null;
+            
+            organizationService.Setup(it => it.Update(organization)).Returns(Task.FromResult(organization));
+
+            // Act
+            var responce = await organizationController.Update(organization);
+
+            // Assert
+            Assert.IsInstanceOf<BadRequestObjectResult>(responce);
+        }
+
+        [Test]
+        public async Task Update_OrganizationExceptionOccurred_ReturnsBadRequestObjectResult()
```

Comment:
in this case the request is valid but user will get BadRequestObjectResult
this might be confusing

consider making changes to return result that describes what happened